os-eap-launch/added/launch/jgroups.sh

```diff
@@ -36,6 +37,20 @@ function configure_jgroups_encryption() {
     else
       log_warning "Partial JGroups encryption configuration, the communication within the cluster WILL NOT be encrypted."
     fi
+  elif [ -n "${JGROUPS_ASYM_ENCRYPT}" ] && [ "${JGROUPS_ASYM_ENCRYPT,,}" == "true" ]; then
```

Comment:
Would it be better to have something like `JGROUPS_ENCRYPT_PROTOCOL` instead, e.g. `JGROUPS_ENCRYPT_PROTOCOL=ASYM_ENCRYPT`?  This would default to `SYM_ENCRYPT` if `JGROUPS_ENCRYPT_NAME` is specified, to preserve backward compatibility.  This would allow you to key of the desired protocol and do whatever is necessary for that protocol.

Reply:
@rcernich Good point / suggestion.

Agree ```JGROUPS_ENCRYPT_PROTOCOL``` is better name. Will update yet (plus change the logic the ```SYM_ENCRYPT``` protocol to be the default one).